cypress/integration/anonymousUser.feature

```diff
@@ -2,9 +2,9 @@ Feature: Anonymous user features
 
   Background:
     Given that there is an existing entity registry with a schema
-
+ 
   Scenario: An anonymous user views an entity without specifying a format
-    Given that there is an entity in the registry
+    And that there is an entity in the registry

```

Comment:
And is in addition to the previous Given?

Reply:
yes